frontend/src/components/header/header.tsx

```diff
@@ -17,29 +32,54 @@ export const Header: React.FC = () => {
     setWishlistCount(wishlist.models.length + wishlist.complectations.length);
   }, [wishlist]);
 
+  const { data } = useGetUserQuery();
+
   const user = {
     favorites: wishlistCount,
     comparisons: 5,
     notifications: 7,
-    avatar: undefined,
+    ...data,
   };
 
   return (
-    <div className={styles.header}>
-      <div className={styles.container}>
-        <Logo />
-        <Navigate />
-      </div>
-      {user ? (
-        <PrivateElements
-          avatar={user.avatar}
-          notifications={user.notifications}
-          comparisons={user.comparisons}
-          favorites={user.favorites}
-        />
-      ) : (
-        <PublicElements />
-      )}
-    </div>
+    <>
+      <AppBar
+        sx={{
+          background: '#ffffff',
+          boxShadow: 0,
+        }}
+      >
+        <Toolbar>
+          <Link to={AppRoute.ROOT}>
+            <img className={styles.logo} src={Logo} alt="Autoline" />
+          </Link>
+          {isMatchSm ? (
+            <DrawerComp />
+          ) : (
+            <>
+              <Tabs
+                onChange={(e, value): void => setValue(value)}
+                value={value}
+                className={styles.nav}
+              >
+                <Tab label="Used Cars" className={styles.navLink} />
+                <Tab label="New Cars" className={styles.navLink} />
+                <Tab label="About us" className={styles.navLink} />
+              </Tabs>
+              {store.getState().auth.token ? (
```

Comment:
Please use `useAppSelector` and avoid usage of the `store.getState()`. It'll make the component rerender on each change of the store